sdk/cognitivelanguage/ai-language-text/README.md

```diff
@@ -38,6 +38,11 @@ Key links:
 
 Please see the [Migration Guide](https://github.com/azure/azure-sdk-for-js/blob/main/sdk/cognitivelanguage/ai-language-text/MIGRATION_ai_text_analytics.md) for detailed instructions on how to update application code from version 5.x of the AI Text Analytics client library to the new AI Language Text client library.
 
+## What's New
+
+For a list of what's new, check out this blog post [What's New in Azure Cognitive Service for Language?](https://learn.microsoft.com/azure/cognitive-services/language-service/whats-new?tabs=javascript) For a quick introduction and tutorial of the new features, see 
+[New beta release of Text Analytics](https://devblogs.microsoft.com/azure-sdk/announcing-the-new-beta-release-of-azure-text-analytics-libraries/).
```

Comment:
Arun's guidance on this is:
> Have a short What's New section linking to recently added samples that you refresh every month or so.

and the blog post has samples only in Python. If we were to follow this guidance, I think we should link to JS samples in the samples folder that show the newly added features.